data/operators/office/government.json

```diff
@@ -70,10 +70,37 @@
         "operator": "Belgische federale regering - Gouvernement fédéral belge - Belgische föderale Regierung"
       }
     },
+    {
+      "displayName": "Bureau of Customs",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["boc"],
+      "tags": {
+        "government": "customs",
+        "office": "government",
+        "operator": "Bureau of Customs",
+        "operator:short": "BOC",
+        "operator:tl": "Kawanihan ng Adwana",
+        "operator:wikidata": "Q25053493"
+      }
+    },
+    {
+      "displayName": "Bureau of Immigration (Philippines)",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["bi", "boi"],
+      "tags": {
+        "government": "immigration",
+        "office": "government",
+        "operator": "Bureau of Immigration",
+        "operator:short": "BOI",
+        "operator:tl": "Kawanihan ng Inmigrasyon",
+        "operator:wikidata": "Q27892670"
+      }
+    },
     {
       "displayName": "Bureau of Internal Revenue",
       "id": "bureauofinternalrevenue-7987d4",
       "locationSet": {"include": ["ph"]},
+      "matchNames": ["bir", "bir revenue district"],
```

Comment:
We don't need to add `bir` to `matchNames`, it's already in other tags (`operator:short`). Anyway, this is not a problem, since the build script will clean unnecessary values. JFYI